vec/vec-v2x/pom.xml

```diff
@@ -27,6 +27,11 @@
             <groupId>jakarta.annotation</groupId>
             <artifactId>jakarta.annotation-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.foursoft.harness.vec</groupId>
+            <artifactId>vec-v12x</artifactId>
+            <version>2.2.0</version>
+        </dependency>
```

Comment:
Why is that needed? And why in that version?

Reply:
removed